capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js

```diff
@@ -77,7 +80,9 @@ function createContentString(name, articles) {
             `<div class="tab-content">
                 <div id="news" class="tab-pane fade in active">
                     <h5><b>Here's the latest news on ${name}:</b></h5>
-                    ${createNewsFeed(articles)}
+
+                    ${createNewsFeed(items)}
+
```

Comment:
im just curious if the blank lines were added intentionally or not

Reply:
yep they were for readability lol